regulations/views/comment.py

```diff
@@ -70,3 +71,29 @@ def submit_comment(request):
         'status': 'submitted',
         'metadata_url': metadata_url,
     })
+
+
+@csrf_exempt
+@require_http_methods(['GET', 'HEAD'])
+def get_federal_agencies(request):
+    url = "{}?{}".format(settings.REGS_GOV_API_LOOKUP_URL,
```

Comment:
I'm thinking it would be simpler to put the base regs.gov API URL in `settings` and keep paths here, like this:

```python
url = os.path.join(settings.REGS_GOV_BASE_URL, 'v3', 'comment.json')
```